test/apicoverage/image/kodata/ignoredfields.yaml

```diff
@@ -98,7 +98,6 @@
     - HostIPC
     - ShareProcessNamespace
     - SecurityContext
-    - ImagePullSecrets
```

Comment:
We should add an integration test :)

Reply:
Yes I agree. I'll create an issue and work on it as a separate PR. I'll also do these for updating the docs.